File: RokidSDKDemoJava/app/src/main/java/com/rokid/mobile/sdk/demo/java/webkit/DemoWebviewActivity.java
package com.rokid.mobile.sdk.demo.java.webkit;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.View;
import android.widget.TextView;

import com.rokid.mobile.sdk.demo.java.R;
import com.rokid.mobile.sdk.demo.java.base.BaseActivity;
import com.rokid.mobile.sdk.demo.java.view.IconTextView;

/**
 * Created by tt on 2018/2/28.
 */

public class DemoWebviewActivity extends BaseActivity {

    public static final String KEY_URL = "url";

    public static final String KEY_TITLE = "title";

    private TextView titleTxt;

    private IconTextView backIcon;

    private DemoWebView webView;

    private String title = "Rokid SDK Demo";

    private String url = "https://www.rokid.com";

    @Override
    protected int getLayoutId() {
        return R.layout.activity_demo_webview;
    }

    @Override
    protected void initVariables(@Nullable Bundle savedInstanceState) {
        title = getIntent().getStringExtra(KEY_TITLE);
        url = getIntent().getStringExtra(KEY_URL);

        titleTxt = findViewById(R.id.webkit_titlebar_title);
        backIcon = findViewById(R.id.webkit_titlebar_left);
        webView = findViewById(R.id.webkit_webview);

        titleTxt.setText(title);
        webView.loadUrl(url);

    }

    @Override
    protected void initListeners() {
        backIcon.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                finish();
            }
        });
    }
}

File: RokidSDKDemoKotlin/app/src/main/kotlin/com/rokid/mobile/sdk/demo/webkit/DemoWebviewActivity.kt
package com.rokid.mobile.sdk.demo.webkit

import com.rokid.mobile.sdk.demo.R
import com.rokid.mobile.sdk.demo.base.BaseActivity
import kotlinx.android.synthetic.main.demo_activity_webview.*

/**
 * Author: Shper
 * Version: V0.1 2018/2/27
 */
class DemoWebviewActivity : BaseActivity() {

    private var title: String = "Rokid SDK Demo"
    private var url: String = "https://www.rokid.com"

    companion object {
        val KEY_URL: String = "url"
        val KEY_TITLE: String = "title"
    }

    override fun layoutId(): Int = R.layout.demo_activity_webview

    override fun initVariable() {
        super.initVariable()

        title = intent.getStringExtra(KEY_TITLE)
        url = intent.getStringExtra(KEY_URL)
    }

    override fun initViews() {
        webkit_titlebar_title.text = title
        webkit_webview.loadUrl(url)
    }

    override fun initListeners() {
        webkit_titlebar_left.setOnClickListener {
            finish()
        }
    }

}